src/isar_robot/robotinterface.py

```diff
@@ -68,29 +68,43 @@ def stop(self) -> bool:
         return True
 
     def get_inspections(self, step: InspectionStep) -> Sequence[Inspection]:
-        if type(step) in [TakeImage, TakeThermalImage]:
-            return self._create_image(step)
-        elif type(step) in [TakeVideo, TakeThermalVideo]:
-            return self._create_video(step)
-        else:
-            return None
+        now: datetime = datetime.utcnow()
+        image_metadata: ImageMetadata = ImageMetadata(
+            start_time=now,
+            time_indexed_pose=TimeIndexedPose(pose=self.pose, time=now),
+            file_type="jpg",
+        )
+        image_metadata.tag_id = step.tag_id
+
+        image: Image = Image(metadata=image_metadata)
+
+        file: Path = random.choice(list(self.example_images.iterdir()))
+
+        with open(file, "rb") as f:
+            data: bytes = f.read()
+
+        image.data = data
+
+        return [image]
```

Comment:
This change I suspect is not yours as it was changed in a previous commit. You need to either rebase on main or if you have resolved some merge conflicts the wrong way you need to fix that.

Reply:
done